resources/tentative/newssite/news-nuxt/nuxt.config.ts

```diff
@@ -0,0 +1,19 @@
+// https://nuxt.com/docs/api/configuration/nuxt-config
+const path = require("path");
+
+const development = process.env.NODE_ENV === "development";
+
+// eslint-disable-next-line no-undef
+export default defineNuxtConfig({
+    ssr: false,
+    css: ["news-site-css/dist/variables.css", "news-site-css/dist/global.css", "news-site-css/dist/a11y.css", "news-site-css/dist/icons.css", "news-site-css/dist/text.css"],
+    components: ["~/components", "~/components/assets", "~/components/atoms", "~/components/molecules"],
+    nitro: {
```

Comment:
this throws an error - I'll look into it
<img width="906" alt="Screenshot 2023-06-02 at 9 45 15 AM" src="https://github.com/WebKit/Speedometer/assets/372973/e8e91980-76bc-4faa-b532-2505acc97da3">

Reply:
update - I "think" this should be correct:

`
 hooks: {
        "pages:extend"(pages) {
            ["home", "us", "world", "politics", "business", "opinion", "health"].map((name) => {
                pages.push({
                    name,
                    path: name === "home" ? "/" : `/${name}`,
                    file: "~/components/molecules/Page.vue",
                });
            });
        },
    },
`

Reply:
You are quite right!

Reply:
Looking at the generate files, using the pages.extend hook does create a new html file for each route. 
The previous solution only created one index.html file, which is what I am hoping for to avoid navigating to a new html page and loading the content during testing.

Reply:
Yes this creates additional files, but we don't use them, we still access just the index, except it also has generated html already.
The individual html files are useful for the web but not for us. But SSR itself could be interesting for the index file. The other files are just byproducts.

Not sure I'm clear :-)

(But as I said above, currently, ssr or not ssr doesn't change anything for us)

Reply:
Note that the index.html file itself is different too, with ssr enabled.

Reply:
You can avoid generating them with a small line of config.

Reply:
oooh that would be nice - do you feel like sharing? 😄

Reply:
I do. Sorry for the bait! Just need to put the kid to bed and I'll provide the abracadabra.

Reply:
💯 - kids first!!